avalanche-ops/src/aws/spec.rs

```diff
@@ -63,7 +63,10 @@ pub struct Spec {
     pub create_dev_machine: bool,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub dev_machine: Option<DevMachine>,
-
+    /// Filepath to an optional directory or file containing bash scripts.
+    /// These scripts will be invoked on the dev machine upon startup.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub dev_machine_scripts_path: Option<Box<std::path::Path>>,
```

Comment:
Although it might be easier for a user to just pass a directory full of scripts. I think when you are executing something it should be explicit. Also handling a file OR directory adds complexity. If you took a comma separated list of files you can also ensure order.

Reply:
Agreed. A list of files sounds like the better API. Will update.